### PR TITLE
feature/SM-7280-Updating api documentation for release sprint 34

### PR DESCRIPTION
**Pull request by maeved**
Signed-off-by: maeved <m.donnelly@kainos.com>

### CODE REVIEW

File: docs/api-documentation/index.html [hunk 1]
```diff
@@ -61,13 +61,13 @@ <h1 class="govuk-heading-m">Releases</h1>
     <li>
       <!-- UPDATE ME to current latest version -->
       <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.6/">
```
Comment (caoimheb) [inline docs/api-documentation/index.html:63]:
```suggestion
      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.7/">
```

File: docs/api-documentation/index.html [hunk 2]
```diff
@@ -61,13 +61,13 @@ <h1 class="govuk-heading-m">Releases</h1>
...
-        Version 3.6 - Released to Sandbox on 22/07/2021. Released to Production on 26/07/2021.
+        Version 3.7 - Released to Sandbox on 05/08/2021. Due for release to Production on 09/08/2021.
       </a>
     </li>
     <li>
       <!-- UPDATE ME to current stable version -->
       <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.16/">
```
Comment (caoimheb) [inline docs/api-documentation/index.html:69]:
```suggestion
      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.17/">
```

File: docs/api-documentation/index.html [hunk 3]
```diff
@@ -82,13 +82,13 @@ <h1 class="govuk-heading-m">Releases</h1>
     <li>
       <!-- UPDATE ME to upcoming latest version -->
       <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.7/">
```
Comment (caoimheb) [inline docs/api-documentation/index.html:84]:
```suggestion
      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V3/V3.8/">
```

File: docs/api-documentation/index.html [hunk 4]
```diff
@@ -82,13 +82,13 @@ <h1 class="govuk-heading-m">Releases</h1>
...
-        Version 3.7 - Due for release to Sandbox on 05/08/2021. Due for release to Production on 09/08/2021.
+        Version 3.8 - Due for release to Sandbox on 19/08/2021. Due for release to Production on 23/08/2021.
       </a>
     </li>
     <li>
       <!-- UPDATE ME to upcoming stable version -->
       <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.17/">
```
Comment (caoimheb) [inline docs/api-documentation/index.html:90]:
```suggestion
      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.18/">
```

**Review by caoimheb — COMMENTED** [review]
Looks good - just a few links to update on the index page

**Review by cmossKainos — APPROVED** [review]

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**